app/views/docs/main.phtml

```diff
@@ -108,7 +108,7 @@ $getSpecsUrl = function($type, $platform) use ($latestVersion) {
     <div class="margin-bottom-large">
         <h2 class="margin-bottom">Learn More</h2>
 
-        <p>Follow our <a href="https://medium.com/appwrite-io" target="_blank" rel="noopener">blog</a> and subscribe to our newsletter to learn more about Appwrite and stay up-to-date with Appwrite latest news and announcements. You can also join our <a href="https://discord.gg/GSeTUeA" target="_blank" rel="noopener">Discord server</a>, where you can chat with fellow developers and get support from our team and community members.</p>
+        <p>Follow our <a href="https://dev.to/appwrite" target="_blank" rel="noopener">blog</a> and subscribe to our newsletter to learn more about Appwrite and stay up-to-date with Appwrite latest news and announcements. You can also join our <a href="https://discord.gg/appwrite" target="_blank" rel="noopener">Discord server</a>, where you can chat with fellow developers and get support from our team and community members.</p>
```

Comment:
```suggestion
        <p>Follow our <a href="https://dev.to/appwrite" target="_blank" rel="noopener">blog</a> and subscribe to our newsletter to learn more about Appwrite and stay up-to-date with Appwrite latest news and announcements. You can also join our <a href="https://appwrite.io/discord" target="_blank" rel="noopener">Discord server</a>, where you can chat with fellow developers and get support from our team and community members.</p>
```

Might be more future proof - since if we loose some boostings on the server, we might loose our custom domain :)